xdem/spstats.py

```diff
@@ -91,6 +90,73 @@ def random_subset(dh: np.ndarray, coords: np.ndarray, nsamp: int):
 
     return dh_sub, coords_sub
 
+def create_circular_mask(h: float, w: float, center: Optional[list[float]] = None, radius: Optional[float] = None) -> np.ndarray:
```

Comment:
Just found out there is a skimage function that does just that! https://www.kite.com/python/docs/skimage.draw.circle

Reply:
Actually `skimage.draw.circle` is being deprecated for `skimage.draw.disk`, which also takes ~3 lines to write because it only outputs coordinates. 
I added the skimage solution too as it's only 3 lines, it behaves slightly differently (interval not inclusive, so radius=2 won't create a circle of 2 pixels, you need to input above, 2.000001 works for instance)